include/pdal/GDALUtils.hpp

```diff
@@ -46,7 +47,7 @@
 #include <vector>
 
 #include <cpl_port.h>
-#include <gdal.h>
+#include <gdal_priv.h>
```

Comment:
We're supposed to avoid including ``gdal_priv.h``. It may be unavoidable though.